server/src/main/scala/drt/server/feeds/acl/AclFeed.scala

```diff
@@ -40,15 +49,15 @@ case class AclFeed(ftpServer: String, username: String, path: String, portCode:
 object AclFeed {
   val log: Logger = LoggerFactory.getLogger(getClass)
 
-  def sftpClient(ftpServer: String, username: String, path: String): SFTPClient = {
+  def sftpClientAndSshClient(ftpServer: String, username: String, path: String): (SFTPClient, SSHClient) = {
     val ssh = new SSHClient()
     ssh.loadKnownHosts()
     ssh.addHostKeyVerifier(new PromiscuousVerifier())
     ssh.connect(ftpServer)
     ssh.authPublickey(username, path)
     ssh.setTimeout(0)
 
-    ssh.newSFTPClient
+    (ssh.newSFTPClient, ssh)
```

Comment:
I'm thinking it might be cleaner to pass an ssh client into this function rather than create one and return the tuple